promotions/developers-salary-survey-2016.adoc

```diff
@@ -0,0 +1,25 @@
+:awestruct-layout: microsite-2
+:awestruct-interpolate: true
+:awestruct-id: microsite-id
+:awestruct-graphic: "http://static.jboss.org/images/rhd/minipage/RHDev_pageimage_developersurvey_11may2016.png"
+
+// Microsite title
+### Curious how you stack up compared to your colleagues? We were curious too.
+
+// Microsite subtitle
+#### So we worked with Evans Data to learn more. Evans Data reached out to 400 enterprise developers to learn what software developers make and what impacts those salary ranges.
+
+
+*In this 2016 Salary Survey Report learn:*
+
+* Developers average salaries
+* How education, company, certifications, and more impact earning potential
+* Why language abilities and developer program membership make a difference
+* 5 key things you can do to increase your income potential
+
+Download the Evans Data Developers Salary Survey Recap to learn more.
+
+Want to dive into the research even further? Download the full report.
+
+[.button]
+link:#{site.download_manager_file_base_url}/download-manager/file/Developers_Salary_Report_2016.pdf[DOWNLOAD NOW]
```

Comment:
Produces:

https://developers.stage.redhat.com/download-manager/file/download-manager/file/Developers_Salary_Report_2016.pdf

Should be:

https://developers.stage.redhat.com/download-manager/file/Developers_Salary_Report_2016.pdf